composables/autoTeleport/useAutoTeleportTransitionDetails.ts

```diff
@@ -214,7 +217,7 @@ export default function (
 
     return [
       hasFetched.teleportTxFee,
-      actionAutoFees.value || hasFetched.actionTxFees,
+      actionAutoFees.value ? hasFetched.actionTxFees : true,
```

Comment:
always return `true` by default?

Reply:
in this case , for nft creation actionTxFees are disabled , and we want to skip that check all together 

can also add `hasFetched.actionTxFees` to an array dynamically if `actionAutoFees.value` is true 

at the end `[].every(Boolean)` would be the same . unless I'm missing something 

Dont wait for `hasFetched.actionTxFees` in this case